hugo/content/4-updates/operators.md

```diff
@@ -0,0 +1,367 @@
++++
+next = "/5-the-bson-mapper/"
+prev = "/4-updates/introduction"
+title = "Operators"
+toc = true
+weight = 1
+
++++
+
+The following is a list of the available **update operators** for the Mangrove query builder.
+This page follows the structure of the
+{{% a_blank "MongoDB update operator documentation" "https://docs.mongodb.com/manual/reference/operator/update/" %}}.
+
+### Field Update Operators
+
+- `{{% a_blank "$inc" "https://docs.mongodb.com/manual/reference/operator/update/inc/" %}}`
+
+    The `$inc` operator is provided as the C++ `+=` and `-=` operators.
+    The unary `++` and `--` can also be used.
+    It is only enabled for fields with a numeric type.
+
+    ```cpp
+    // Increment the age of users who have a birthday today.
+    // The following three statements are the same, and produce the BSON:
+    // { $inc: {age: 1}}
+    auto res = collection.update_many(MANGROVE_KEY(User::is_birthday) == true,
```

Comment:
For all of these, just to make it more readable, what do you think about having an empty match expression?
e.g.
```c++
auto res = collection.update_many({}, MANGROVE_KEY(User::age) += 1);